### DISCUSSION TITLE
Are we even using this file

### DISCUSSION THREAD

**Discussion by aidan-crockett** [theforumhelpers/theforumhelpers.github.io#174] [Q&A] [answered]
Are we even using [coder.png](https://github.com/theforumhelpers/theforumhelpers.github.io/blob/master/resources/coder.png)?

The only usage I found with vscode's "search all files" feature is this:
https://github.com/theforumhelpers/theforumhelpers.github.io/blob/08fe05b5f38fc4109697c9b6c35922fb57a258a0/forumhelpers/index.html#L29-L31

I figured this wasn't worthy of being an issue 😅

**Comment by Accio1** [comment, marked as answer]
@gosoccerboy5 it was previously used to indicate when someone helped code the site. The code that referenced it was commented out in https://github.com/theforumhelpers/theforumhelpers.github.io/commit/eacff430e0c7fee6932119035ac2add25145ebe4 when the site was reworked to use JSON. I left the file in pending re implementing it on the page, but mostly forgot that it was there. Adding badges for people who helped code the site was considered as a part of #36 but wasn't implemented in favour of a contributors page.
